cli/src/main/java/com/devonfw/tools/ide/commandlet/BuildCommandlet.java

```diff
@@ -47,45 +45,33 @@ public String getName() {
   public void run() {
 
     Path buildPath = this.context.getCwd();
-    String[] defaultToolOptions = new String[0];
 
     if (buildPath == null) {
       throw new CliException("Missing current working directory!");
     }
 
-    ToolCommandlet commandlet = null;
-    if (Files.exists(buildPath.resolve("pom.xml"))) {
-      commandlet = this.context.getCommandletManager().getCommandlet(Mvn.class);
-      defaultToolOptions = getDefaultToolOptions(MVN_BUILD_OPTS.getName());
-    } else if (Files.exists(buildPath.resolve("build.gradle"))) {
-      commandlet = this.context.getCommandletManager().getCommandlet(Gradle.class);
-      defaultToolOptions = getDefaultToolOptions(GRADLE_BUILD_OPTS.getName());
-    } else if (Files.exists(buildPath.resolve("package.json"))) {
-      if (Files.exists(buildPath.resolve("yarn.lock"))) {
-        // TODO: add yarn here
```

Comment:
it is always better to create an issue instead of a code TODO.
Things like that simply got forgotten.